### PR TITLE
Fucking hell

### PR DESCRIPTION
**Pull request by Keyinator**
Added command for patreon-reps to add/remove patreon-supporter role

### CODE REVIEW

**Review by Danstr5544 — APPROVED** [review]
Seems Fine to me, something something, snarky comment that for once I can approve something

**Comment by AgentBlackout** [comment]
Targeted comment much? @Danstr5544

**Comment by sirmeepington** [comment]
Cringe

File: NWTools/nwtools.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+from discord.utils import get
+from redbot.core import commands, Config, checks
+from redbot.core.bot import Red
+from typing import Optional
+import asyncio
+
+"""
+I fucking swear to god that the fact that I need to make this cog a thing because I can't be 100% certain that folks
+won't just fucking massmute folks in the server is the only joke thats bigger than this cogs required existence.
+I WILL NOT be adding proper variable support, and WILL be hardcoding ID's.
```
Comment (sirmeepington) [inline NWTools/nwtools.py:11]: pog

File: NWTools/nwtools.py [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+I WILL ONLY allow this cog to function within the SL Discord.
```
Comment (sirmeepington) [inline NWTools/nwtools.py:12]: not pog

File: NWTools/nwtools.py [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+I WILL tell *anyone* that tells me that this cog isn't up to par, Including Masonic (tho frankly he's probably read this and is just staying away from it)
+To go fuck themselves on a bed of rusty fucking nails - because if I have to fuck my evening over due to trust issues, then fuck you too.
+
+See now you can even read it nicely
```
Comment (sirmeepington) [inline NWTools/nwtools.py:16]: thanks love xoxo

**Review by sirmeepington — COMMENTED** [review]
nice

**Comment by Keyinator** [comment]
> Cringe

Are you hearting your own comment?

Cringe xD

**Pull request merged**